samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs

```diff
@@ -179,10 +175,94 @@ private IEnumerator PipelineChanged()
 
         yield return null;
 
-        SetPipeline(PipelineTypeDropdown.options[PipelineTypeDropdown.value].text);
+        // Set the Pipeline based on what Pipeline button is clicked
+        SetPipeline(PipelineModeText);
+
+        SceneButtonOnClick();
+    }
+
+    public void PipelineButtonOnClick()
```

Comment:
You have a mix of `public` and `private` functions, they should be grouped, all `public` functions above `private` functions, and each group of function should be ordered in an way that makes logical sense, like alphabetical or the order/sequence of how its called